tests/browser_test_hello_world.c

```diff
@@ -25,7 +25,9 @@ int main() {
     if (Module['prints'].length !== 1) throw 'bad length ' + Module['prints'].length;
     if (Module['prints'][0] !== 'hello, world!') throw 'bad contents: ' + Module['prints'][0];
   });
+#ifdef REPORT_RESULT
   REPORT_RESULT(0);
```

Comment:
If REPORT_RESULT was not defined then this would fail to build...  which makes me think it is always defined.. which makes me think this change is not needed.

Reply:
I think the `REPORT_RESULT` function exists whether or not the `REPORT_RESULT` flag is set. If you do a project wide find for "#ifdef REPORT_RESULT" you can see the 275 other places that have this check in place. This fixture just had the assumption that the report should always happen.